sql/atxs/atxs.go

```diff
@@ -15,7 +15,7 @@ func Get(db sql.Executor, id types.ATXID) (atx *types.VerifiedActivationTx, err
 		stmt.BindBytes(1, id.Bytes())
 	}
 	dec := func(stmt *sql.Statement) bool {
-		var v types.VerifiedActivationTx
```

Comment:
there should be a test that failed before your fix but passed with this.
i.e. 
- add a atx to db
- get the blob out
- parse it correctly

Reply:
There is a test in `store_test.go` that asserts this: `TestBlobStore_GetATXBlob`

Reply:
but why didn't it fail when systest failed when the wrong type was parsed?